x-pack/plugins/data_visualizer/public/application/file_data_visualizer/components/edit_flyout/overrides.js

```diff
@@ -224,8 +236,17 @@ class OverridesUI extends Component {
     this.setOverride({ columnNames });
   };
 
+  onGrokPatternFieldChange = (e, i) => {
```

Comment:
Also applies to editing fields for CSV files, but would be good to have some basic validation on the new field name e.g. that it isn't the same as another field name. One for a follow-up perhaps?

Reply:
Yes, it would be good to have some validation on the entered fields. I think this is outside of the scope of this PR and can be done in a follow up.